packages/preferences/src/browser/views/preference-editor-widget.ts

```diff
@@ -101,7 +101,7 @@ export class PreferencesEditorWidget extends BaseWidget implements StatefulWidge
         } else {
             ((otherSource: never) => { throw new Error('You should handle all possible sources of change!'); })(e.source);
         }
-        this.resetScroll(currentFirstVisible, !isFiltered);
+        this.resetScroll(currentFirstVisible, e.source === PreferenceFilterChangeSource.Search && !isFiltered);
```

Comment:
Fixes the issue of scrolling to top when switching scopes.

Reply:
Verified that this fixes it, nice job